### PR TITLE
add and remove pets from saved list

### PR DESCRIPTION
**Pull request by ctedtsen**
Allows signed in user to add pets to their personal saved list (they can also remove pets). I also added a page to sho the users saved lists.

### CODE REVIEW

**Comment by ctedtsen** [comment]
I have also updated it so that signed in users can add/remove shelters from their saved list.

File: src/routes/users.js [hunk 1]
```diff
@@ -115,6 +115,29 @@ router.route("/logout").get(async (req, res) => {
...
+  }
+  let user;
+  try{
+      user = await userData.findByUsername(loggedIn);
+  } catch(e) {
+    res.redirect("/users/login");
+    return;
+  }  
+  try{
+      res.render('profile/saved', {title: "Saved Shelters and Pets", user: user});
```
Comment (Jasmine211) [inline src/routes/users.js]: Can you add the loggedIn field for rendering on line 135?

**Review by Jasmine211 — CHANGES_REQUESTED** [review]
Thank you, this looks good. Just fix one small thing in routes

File: src/routes/users.js (hunk 1 above)
Comment (ctedtsen) [reply src/routes/users.js]: Yup! just added it!

**Review by Jasmine211 — APPROVED** [review]
Looks good!

**Pull request merged**